Automattic/newspack-blocks
# Carousel: handle posts without featured images

### All Submissions:






### Changes proposed in this Pull Request:



Closes #443

### How to test the changes in this Pull Request:

1. Insert three Carousel blocks on a page: one that renders posts with featured images, one that renders posts without any, and one which would render no posts
2. In the editor, observe the first is behaving as expected, the second is displaying grey surface in place of the featured image, and the last is displaying a center-aligned error message (and no carousel arrows)
3. On frontend, first two should behave as in the editor, while the last should not render any visible content

### Other information:
